### ISSUE TITLE
Mission objectives of Tropica 3 (On the Offensive) is misleading

### ISSUE THREAD

**Issue by gergelypolonkai** [colobot/colobot-data#73]
SatCom tells me to terminate all life forms, but mission ends when I launch the space ship. Maybe this fact should be included in the objectives?

**Comment by Zemogiter** [comment]
We should close this issue, since it's been fixed.

**Comment by hexagonrecursion** [comment]
No. It has not been fixed. There is still no mention of having to press the "Take off" button in the Tropica - On the Offensive help file

> From Mission Control
> Terminate all hostile life-form.
> Perform the necessary research program in order to develop an offensive weapon system.
> 
> The satellite has detected an energy source, on a spot marked by the previous expedition with a blue flag (blue cross on the mini-map).
> 
> Do not forget to take a look at the satellite report before you venture any further in a mission; we suspect that you have not always done this.
> 
> We have improved the recharge program. Please refer to the page
> 
> A note of warning: bots cannot be submerged. If in doubt, test the water yourself if you plan to radio-control a non-winged bot across - water shouldn't be deeper than knee-level.
> 
> F1 allows you to review these instructions at all times on your personal SatCom
> 
> See also
> Controls and CBOT Language
